integration-webhook/src/main/java/org/symphonyoss/integration/webhook/parser/metadata/MetadataParser.java

```diff
@@ -233,8 +240,15 @@ private void processMetadataObjects(EntityObject root, JsonNode node, List<Metad
         }
       }
 
-      if (object.getChildren() != null) {
-        processMetadataObjects(entity, node, object.getChildren());
+      if (object.getChildren() != null || object.isList()) {
```

Comment:
Could it be?

```
if (object.isList()) {
  List itemsList = processListMetadataObjects(node, object, entity);
  root.addContent(object.getId(), itemsList);
} else if (object.getChildren() != null) {
  processMetadataObjects(entity, node, object.getChildren());
}
```

You could also pass the EntityObject root as a parameter and include the result to the root within the method instead of returning a list.